src/schema/formats.yaml

```diff
@@ -0,0 +1,88 @@
+---
+# This file defines valid patterns for different formats
+# Entity patterns
+label:
+  description: |
+    Freeform labels without special characters.
+  pattern: ^[0-9a-zA-Z]+$
+index:
+  description: |
+    Non-negative, non-zero integers, optionally prefixed with leading zeros for sortability.
+    An index may not be all zeros.
+  pattern: ^[0-9]*[1-9]+[0-9]*$
+# Metadata types
+string:
+  description: |
+    The basic string type (not a specific format).
+    This should allow any free-form string *except* "n/a".
+  pattern: ^(?!(n/a)$).*$
```

Comment:
This is related to #876 

> This should allow any free-form string *except* "n/a".

I am not sure whether disallowing `n/a` as a string for free-form text fields would break some datasets. For example for the required `EEGReference` field, users could currently pass `n/a` according to bids-validator:

https://github.com/bids-standard/bids-validator/blob/75c54e4f7a17bf58c7cfc47c9698eb4e0841315b/bids-validator/validators/json/schemas/eeg.json#L25

Whether or not that's a good idea should be discussed in #876, this is just a note that what you introduce here has not been discussed and has not been formally part of the spec :thinking: